src/App.js

```diff
@@ -0,0 +1,37 @@
+import { Routes, Route } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import Home from './components/Home/Home';
+import Cars from './components/pages/Cars';
+import Login from './components/Registration/Login';
+import SignUp from './components/Registration/SignUp';
+import NewCar from './components/NewCar/NewCar';
+import NewReservation from './components/NewReservation/NewReservation';
+import Details from './components/pages/Details';
+import ReservedCars from './components/Reservation/reservedCars';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this function in your project to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.